tests/test_strategies.py

```diff
@@ -281,8 +281,7 @@ def test_only_check_game_end_once(self):
         self.assertEqual(player.root.child_N[pass_move], 1)
 
     def test_extract_data_normal_end(self):
-        player = MCTSPlayer(DummyNet())
-        player.initialize_game()
+        player = initialize_basic_player()
         player.tree_search()
```

Comment:
re: above, this would have searched twice?

Reply:
yeah, I think this isn't a functionally equivalent substitution.

Reply:
This test doesn't really need to invoke search, and invoking it twice isn't a bad thing.
it's about making sure that is_done() get's set when you call play_move(None) twice.